manifests/3/3ll3d00d/beqdesigner/1.1.8/3ll3d00d.beqdesigner.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with komac v2.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: 3ll3d00d.beqdesigner
+PackageVersion: 1.1.8
+InstallerType: portable
+ReleaseDate: 2023-10-21
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/3ll3d00d/beqdesigner/releases/download/1.0.1/beqdesigner_10.exe
+  InstallerSha256: 6BEFC8983A76968670CC5664A62539342D1AFA771334524ADF98EB31117A5813
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/3ll3d00d/beqdesigner/releases/download/1.0.1/beqdesigner_10.exe

```

Comment:
These `InstallerUrl`s appear to contain a previous version number, instead of the [release](https://github.com/3ll3d00d/beqdesigner/releases/tag/1.1.8) for the `PackageVersion`.